pkg/steps/release/release_images.go

```diff
@@ -214,7 +215,10 @@ func (s *releaseImagesTagStep) Requires() []api.StepLink {
 }
 
 func (s *releaseImagesTagStep) Creates() []api.StepLink {
-	return []api.StepLink{api.ReleaseImagesLink()}
+	return []api.StepLink{
+		api.StableImagesLink(api.InitialStableName),
```

Comment:
Technically this is not a no-op ... but we always created both streams here, we just never exposed the `stable-initial` one nor could anyone require it.